flume/datadog_checks/flume/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
You can set the version to 1.0.0 if you want, that'll be the initial version we'll release.